packages/web/src/scss/components/Footer/index.html

```diff
@@ -0,0 +1,480 @@
+            <h3 class="typography-body-large-text-bold mb-600" id="footer-part-two">
+              Section headline
+            </h3>
+            <ul class="Stack Stack--hasSpacing" style="--stack-spacing: var(--spirit-space-500)">
+              <li>
+                <a href="https://www.example.com">Link</a>
+              </li>
+              <li>
+                <a href="https://www.example.com">Link</a>
+              </li>
+              <li>
+                <a href="https://www.example.com">Link</a>
+              </li>
+              <li>
+                <a href="https://www.example.com">Link</a>
+              </li>
+            </ul>
+          </nav>
+
+          <!-- Navigation links 3 -->
+          <nav aria-labelledby="footer-part-three">
+            <h3 class="typography-body-large-text-bold mb-600" id="footer-part-three">
+              Section headline
+            </h3>
+            <ul class="Stack Stack--hasSpacing" style="--stack-spacing: var(--spirit-space-500)">
+              <li>
+                <a href="https://www.example.com">Link</a>
+              </li>
+              <li>
+                <a href="https://www.example.com">Link</a>
+              </li>
+              <li>
+                <a href="https://www.example.com">Link</a>
+              </li>
+              <li>
+                <a href="https://www.example.com">Link</a>
+              </li>
+              <li>
+                <a href="https://www.example.com">Link</a>
+              </li>
+              <li>
+                <a href="https://www.example.com">Link</a>
+              </li>
+            </ul>
+          </nav>
+
+          <!-- Navigation links 4 -->
+          <nav aria-labelledby="footer-part-four">
+            <h3 class="typography-body-large-text-bold mb-600" id="footer-part-four">
+              Section headline
+            </h3>
+            <ul class="Stack Stack--hasSpacing" style="--stack-spacing: var(--spirit-space-500)">
+              <li>
+                <a href="https://www.example.com">Link</a>
+              </li>
+              <li>
+                <a href="https://www.example.com">Link</a>
+              </li>
+              <li>
+                <a href="https://www.example.com">Link</a>
+              </li>
+            </ul>
+          </nav>
+
+        </div>
+
+        <!-- Divider -->
+        <hr class="Divider my-900" />
+
+        <!-- Grid with product logo, social media links and language switch -->
+        <div
+          class="Grid Grid--cols-1 Grid--desktop--cols-3 Grid--alignmentXCenter Grid--desktop--alignmentXStretch Grid--alignmentYCenter"
+          style="--grid-spacing-x: var(--spirit-space-900); --grid-spacing-y: var(--spirit-space-900)"
+        >
+
+          <!-- Product logo -->
+          <div class="text-desktop-left">
+            <a href="https://www.example.com">
+              <div aria-label="Logo of the JobBoard" class="UNSTABLE_ProductLogo">
```

Comment:
https://www.w3.org/TR/using-aria/#practical-support-aria-label-aria-labelledby-and-aria-describedby

Aria-label is probably not working without a role.

> Don't use aria-label or aria-labelledby on a span or div unless its given a role. When aria-label or aria-labelledby are on interactive roles (such as a link or button) or an img role, they override the contents of the div or span. Other roles besides Landmarks (discussed above) are ignored.

Reply:
Hmm, this means the `ProductLogo` component needs to be fixed too…

Reply:
Addressed in #1651.

Reply:
Cool.